### ISSUE TITLE
Unable to Run Example: MissingPluginException

### ISSUE THREAD

**Issue by AkanshDivker** [flutter-institute/webauthn#1]
I'm trying to get started with this package but I'm unable to get the example running. After cloning the repository, I ran the tests and they all passed. However, when I try and run the example project I get a ```MissingPluginException``` when trying to register. I have tried running ```flutter clean``` before running the example but the issue still persists. I hope I'm not missing any steps or instructions, I believe all compatibility requirements are being met.

**Platform:** Windows 11
**Error:** MissingPluginException (MissingPluginException(No implementation found for method getDatabasesPath on channel com.tekartik.sqflite))

Would appreciate any help, thanks!

**Console Log**

```
[ERROR:flutter/runtime/dart_vm_initializer.cc(41)] Unhandled Exception: MissingPluginException(No implementation found for method getDatabasesPath on channel com.tekartik.sqflite)
#0      MethodChannel._invokeMethod (package:flutter/src/services/platform_channel.dart:294:7)
<asynchronous suspension>
#1      wrapDatabaseException (package:sqflite/src/exception_impl.dart:7:20)
<asynchronous suspension>
#2      SqfliteDatabaseFactoryMixin.getDatabasesPath (package:sqflite_common/src/factory_mixin.dart:151:20)
<asynchronous suspension>
#3      DB._getDirectory (package:webauthn/src/db/db.dart:64:29)
<asynchronous suspension>
#4      DB._getDbPath (package:webauthn/src/db/db.dart:60:17)
<asynchronous suspension>
#5      DB.open (package:webauthn/src/db/db.dart:44:12)
<asynchronous suspension>
#6      DBSchema.execute (package:webauthn/src/db/schema.dart:38:20)
<asynchronous suspension>
#7      CredentialSchema._getOne (package:webauthn/src/db/credential.dart:194:21)
<asynchronous suspension>
#8      Authenticator.makeCredential (package:webauthn/src/authenticator.dart:115:13)
<asynchronous suspension>
#9      _MyHomePageState._createCredential.<anonymous closure> (package:webauthnexample/main.dart:128:29)
<asynchronous suspension>
#10     _MyHomePageState._lockAndExecute (package:webauthnexample/main.dart:98:18)
<asynchronous suspension>
#11     _MyHomePageState._createCredential (package:webauthnexample/main.dart:117:26)
<asynchronous suspension>
```

**Comment by AkanshDivker** [comment]
Thanks for the information! I did some more digging and found out that for Linux and Windows some extra initialization code is necessary in the main method. I've created a pull request which includes a fix, hope it's all fine and good!

#2

**Comment by killermonk** [comment]
Have merged PR #2. Thank you so much!
